### PR TITLE
Remove drwill-ms as a codeowner on modelsrepository

### PR DESCRIPTION
**Pull request by drwill-ms**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by jsquire — APPROVED** [review]

**Comment by drwill-ms** [comment]
I'll fix the conflicts.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -280,7 +280,7 @@
 /sdk/communication/Azure.ResourceManager.Communication/            @archerzz @ArcturusZhang @ArthurMa1978
 
 # PRLabel: %Communication - Rooms
-/sdk/communication/Azure.Communication.Rooms/                      @minnieliu @paolamvhz @alexokun @Mrayyan @shwali-msft @allchiang-msft @mikehang-msft
+/sdk/communication/Azure.Communication.Rooms/                      @minnieliu @paolamvhz @alexokun @Mrayyan @shwali-msft @allchiang-msft�@mikehang-msft
```
Comment (drwill-ms) [inline .github/CODEOWNERS:283]: Not sure how this happened. Fixing.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -280,7 +280,7 @@
 /sdk/communication/Azure.ResourceManager.Communication/            @archerzz @ArcturusZhang @ArthurMa1978
 
 # PRLabel: %Communication - Rooms
-/sdk/communication/Azure.Communication.Rooms/                      @minnieliu @paolamvhz @alexokun @Mrayyan @shwali-msft @allchiang-msft @mikehang-msft
+/sdk/communication/Azure.Communication.Rooms/                      @minnieliu @paolamvhz @alexokun @Mrayyan @shwali-msft @allchiang-msft @mikehang-msft
```
Comment (drwill-ms) [inline .github/CODEOWNERS:283]: I didn't wish to make a change, but this showed up as a foreign character, so I replaced it with a space.

**Comment by drwill-ms** [comment]
Ready to merge, @jsquire. CheckEnforcer is pending which IIRC always happens when a change is only to this file.

**Pull request merged**